ilastik/applets/objectClassification/objectClassificationGui.py

```diff
@@ -175,7 +175,14 @@ def __init__(self, parentApplet, op):
         self.labelingDrawerUi.checkShowPredictions.toggled.connect(
             self.handleShowPredictionsClicked)
 
-        #select all the features in the beginning
+        # Add two permanent labels because it makes no sense to have less here
+        self._addNewLabel()
+        self._addNewLabel()
+        self.labelingDrawerUi.labelListModel.makeRowPermanent(0)
+        self.labelingDrawerUi.labelListModel.makeRowPermanent(1)
+        self.labelingDrawerUi.labelListView.shrinkToMinimum()
```

Comment:
I think this line kind of breaks the layout for me. Same holds true for Object Classification.

* with this PR
![pc-new](https://user-images.githubusercontent.com/24434157/41529995-10cebf88-72ef-11e8-91c4-704ca9897eb2.png)
* old state
![pc-old](https://user-images.githubusercontent.com/24434157/41530016-1f4e78be-72ef-11e8-842d-b2091876f0c9.png)

could you please make it behave it like in the lower screenshot?